internal/output/outputter.go

```diff
@@ -0,0 +1,147 @@
+package output
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/windmilleng/tilt/internal/logger"
+)
+
+type Color string
+
+const (
+	cGreen                = Color("\033[32m")
```

Comment:
could you use a color library like https://github.com/fatih/color?

most good color libraries will try to detect if you have a terminal that supports color, so that we don't end up printing color codes to terminals that don't understand them